modules/popup/css/lib/popup.scss

```diff
@@ -1,203 +1,120 @@
-.wdc-popup {
-  @include wdc-type();
-  @include wdc-type-body();
-  @include wdc-depth-3();
-  display: inline-block;
-  background-color: $_wdc-popup-bg-color;
-  padding: $wdc-spacing-s;
-  border-radius: 3px;
-  border: 1px solid $_wdc-popup-border-color;
-  position: relative;
-  width: max-content;
-  max-width: $_wdc-popup-max-width;
-
-  // Fallback for IE11
-  @media screen and (-ms-high-contrast: active),
-  screen and (-ms-high-contrast: none) {
-    width: auto;
-    min-width: $_wdc-popup-menu-min-width;
+@keyframes popupAnimationTopCenter {
+  from {
+    opacity: 0;
+    transform: translate(0, -50%);
```

Comment:
We should only be translating `spacing.xxs` (https://github.com/Workday/canvas-kit/blob/master/modules/popup/react/lib/Popup.tsx#L31)